apps/parse/lib/parse/vehicle_positions_json.ex

```diff
@@ -47,6 +48,13 @@ defmodule Parse.VehiclePositionsJson do
     []
   end
 
+  defp parse_consist([_ | _] = consist) do
+    consist
+    |> Enum.map(fn %{"label" => car_label} -> car_label end)
```

Comment:
I'm surprised this doesn't get caught by Credo: shouldn't use a pipe for a single function call.